### ISSUE TITLE
[Action Bar] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1417]
### Feedback

- Action bar is in the DOM (not hidden) even when not visible. This will cause screen readers to read it even when not displayed visually

### Path forward

1. Interest, most of the other elements use `display: none` in this case, which prevents this (right?). We can sprout `aria-hidden="true"` when `open === false` and we should be set here.

cc @jnurthen
